task_geo/data_sources/hdx_oxfd/__init__.py

```diff
@@ -0,0 +1,21 @@
+from task_geo.data_sources.hdx_oxfd.hdx_oxfd_connector import hdx_oxfd_connector
+from task_geo.data_sources.hdx_oxfd.hdx_oxfd_formatter import hdx_oxfd_formatter
+
+
+def hdx_oxfd(start_date=None, end_date=None, country_iso2=None):
+    """Retrieves formatted Oxford COVID-19 Government Response Tracker Dataset from HDX DataSource
+    Arguments:
+        start_date(str) : YYYY-MM-DD ISO Format
```

Comment:
Dates should be `datetime.date` or `datetime.datetime` then transformed to the format inside the function.